proto/dwtools/abase/l0.test/Buffer.test.s

```diff
@@ -10770,6 +10770,72 @@ function bufferJoin( test )
 
 //
 
+function bufferMove( test )
+{
+
+  test.case = 'dst - array, src - array';
+  var dst = [ 1, 2, 3, 4 ];
+  var src = [ 1, 2, 3, 4 ];
+  var got = _.bufferMove( dst, src );
+  test.identical( got, src );
+  test.is( got !== src );
+
+  test.case = 'dst - array, src - U8x';
+  var dst = [ 0, 2, 3 ];
+  var src = new U8x( [ 1, 2, 3 ] );
+  var got = _.bufferMove( dst, src );
+  var expected = [ 1, 2, 3 ];
+  test.identical( got, expected );
+
+  test.case = 'dst - array, src - U16x';
+  var dst = [ 0, 2, 3 ];
+  var src = new U16x( [ 1, 2, 3 ] );
+  var got = _.bufferMove( dst, src );
+  var expected = [ 1, 2, 3 ];
+  test.identical( got, expected );
+
+  test.case = ' identical dst - U16x, src - I16x';
+  var dst = new U16x( [ 1, 2, 3 ] );
+  var src = new I16x( [ 1, 2, 3 ] );
+  var got = _.bufferMove( dst, src );
+  var expected = new U16x( [ 1, 2, 3 ] );
+  test.identical( got, expected );
+
+  test.case = 'dst - I16x, src - U16x';
+  var dst = new I16x( [ 1, 2, 3 ] );
+  var src = new U16x( [ 4, 5, 6 ] );
+  var got = _.bufferMove( dst, src );
+  var expected = new I16x( [ 4, 5, 6 ] );
+  test.identical( got, expected );
+
+  test.case = 'dst - U16x, src - I16x';
+  var dst = new U16x( [ 4, 5, 6 ] );
+  var src = new I16x( [ 1, 2, 3 ] );
+  var got = _.bufferMove( dst, src );
+  var expected = new U16x( [ 1, 2, 3 ] );
+  test.identical( got, expected );
```

Comment:
Please, implement test routine `bufferMoveWithSingleArgument`.

This test routine should have similar test cases and next extension:
dst - BufferRaw
Also, use option `offset`.

Example of test case:
```js
  test.case = 'dst - BufferRaw, src - array, offset - 2';
  var dst = new BufferRaw( 6 );
  var src = [ 1, 2, 3, 4 ];
  var got = _.bufferMove( { dst : dst, src : src, offset : 2 } );
  var expected = new U8x( [ 0, 0, 1, 2, 3, 4 ] ); // I can make mistake. It may be new U8x( [ 1, 2, 3, 4 ] );
  test.identical( got, expected );
  test.is( got.buffer === dst ); // it is important. 
  test.is( got !== dst ); // for other buffers should be ( got === dst )
  test.is( got !== src );
```